server.ts

```diff
@@ -0,0 +1,57 @@
+import * as express from 'express';
+import { TodoApp } from './src/todo';
+
+const todo = new TodoApp();
+
+const serv = express();
+serv.use(express.json());
+
+const port = 3000;
+
+serv.get('/', function (req, res) {
+    res.send(todo.listAllToDos());
+});
+
+serv.put('/', function (req, res) {
+    const data = req.body;
+    try {
+        todo.addItem(data.description, data.dueAt);
+        res.sendStatus(200);
+    } catch (e) {
```

Comment:
This works but is very redundant and painful to type I imagine.  There are better ways to do this.  Did you see any while reading the express docs?  If you can't think of any we can chat about it and I can point you in a direction to make this much easier.

Reply:
I've updated it to use some express error handling middleware for now.

Reply:
I see that.  Of note you actually don't even need the try/catch here as the function in Express that calls your anonymous function will handle the try/catch.  So you could even remove those and still have the same result.